google-cloud-clients/google-cloud-vision/src/main/java/com/google/cloud/vision/v1/ProductSearchClient.java

```diff
@@ -127,129 +126,6 @@
   private final ProductSearchStub stub;
   private final OperationsClient operationsClient;
 
-  private static final PathTemplate LOCATION_PATH_TEMPLATE =
-      PathTemplate.createWithoutUrlEncoding("projects/{project}/locations/{location}");
-
-  private static final PathTemplate PRODUCT_PATH_TEMPLATE =
-      PathTemplate.createWithoutUrlEncoding(
-          "projects/{project}/locations/{location}/products/{product}");
-
-  private static final PathTemplate PRODUCT_SET_PATH_TEMPLATE =
-      PathTemplate.createWithoutUrlEncoding(
-          "projects/{project}/locations/{location}/productSets/{product_set}");
-
-  private static final PathTemplate IMAGE_PATH_TEMPLATE =
-      PathTemplate.createWithoutUrlEncoding(
-          "projects/{project}/locations/{location}/products/{product}/referenceImages/{image}");
-
-  /** Formats a string containing the fully-qualified path to represent a location resource. */
-  public static final String formatLocationName(String project, String location) {
```

Comment:
@andreamlin Why is the generator removing these? This API is GA so these are breaking changes.

Reply:
ack

Reply:
It might be https://github.com/googleapis/gapic-generator/commit/c76948c11637104523180ec04d0754c84065c900; I'll try locally rolling it back to see what happens.